RELEASE.md

```diff
@@ -1,53 +1,29 @@
 # Release Process
 
-## Automatic release (preferred)
-
-The release process consists of these steps:
-  1. Create a pull request with:
-     * Change the version in [`build.gradle`](build.gradle) to match the release version e.g. `0.20.0`
-     * Add an entry to [`CHANGELOG`](CHANGELOG.md) with changes since the last release
-  1. Commit your `CHANGELOG` changes
-  1. Create and push tag with the new version `git tag v0.20.0 && git push origin v0.20.0`
-  1. Once the *tag* build finishes in Travis, the artifacts should have been uploaded to Sonatype staging,
-     the staging repository closed, and the artifacts on the way to Maven Central (it takes 20min+ to get there).
-     In case of failures it is safe to retry by restarting the maven-deploy job. If it keeps
-     failing, sometimes it may be necessary to close the staging repository manually.
-  1. Create a release on GitHub for the new tag. Use the changes from the `CHANGELOG` as the description.
-  1. Once the artifacts are available on Maven
-     * bump the version in `build.gradle` to `major.minor.(patch+1)-SNAPSHOT`
-     * add a new section to the CHANGELOG with that version as `(unreleased)` and a bullet point `- Nothing yet`
-     * commit with the comment "Back to development" (for example, https://github.com/jaegertracing/jaeger-client-java/commit/da9726d3ba7309947882e3c621516c70b4bc83dc)
-
-## Local setup and release
-
-The Travis build execute the `uploadArchives` task, and this task performs the signing of the artifacts.
-While the Travis build is ready for that, releasing locally requires the following steps:
-
- * install GPG: `brew install gpg`
- * generate keys: `gpg --gen-key`
- * see installed keys with `gpg --list-keys` or `gpg --list-secret-keys`
- * create an account with Sonatype, get repo permissions (e.g. https://issues.sonatype.org/browse/OSSRH-23572)
- * create `$HOME/.gradle/gradle.properties` file or add parameters directly as it is shown at the last step.
-```
-    signing.keyId={key ID from gpg --list-secret-keys}
-    signing.password={password you used to encrypt keys via gpg --gen-key}
-    signing.secretKeyRingFile={e.g. [home path]/.gnupg/secring.gpg}
+This project uses [Shipkit](http://shipkit.org) for most of the release management. A release can be performed either manually or via Travis.
+
+## Automated
+
+1. Update the `CHANGELOG.md` with the changes for the last release. Most of the changes should have been added there already, but at least the release date has to be set.
+1. Get the change above merged
+1. Tag the release, like, `git tag release/v0.31.1` and push the tag to the main repository (`git push upstream v0.31.0`, if your `jaegertracing/jaeger-client-java` remote is called `upstream`)
```

Comment:
Could we keep the same tag names? So without `release/`?

Reply:
If there are intermediate commits required, like when bumping from 0.0.2 to 1.0.0, then we should do it in the `release/` tag build, otherwise our `v1.0.0` tag would have `version.properties` as `0.0.2`.

Reply:
> Does the release process crate a tag? So when I push release/vx.x.x does it create vx.x.x?

Yes, when you push `release/v0.0.1`, it creates the `v0.0.1` tag.

> Could you please explain it for both cases what tags are created when there is a change and when there is no change required in version file?

Sure, I'll update the readme.

Reply:
Readme updated